docs/INSTALL.MD

```diff
@@ -1,18 +1,90 @@
-## Installation
+# Installation
 
 JuliaBox is developed and tested on Ubuntu 14.04. But it should be easy to adapt to any Linux environment supporting Docker.
 
-### Single Machine Development Setup
-1. Get JuliaBox sources:
-    - If `git` is not installed yet, run: `sudo apt-get install git`
-    - `git clone https://github.com/tanmaykm/JuliaDockerImages.git`
-    - `git clone https://github.com/JuliaLang/JuliaBox.git`
-2. Install docker and a few more packages required to develop and administer JuliaBox:
-    - Modify `JuliaBox/scripts/install/sys_install.sh` and set `DOCKER_FS` to `AUFS` instead of `DEVICEMAPPER`.
-        - This will avoid certain unnecessary configurations required only in certain production environments.
-    - Run `JuliaBox/scripts/install/sys_install.sh`
-3. Create work folders: `mkdir -p /jboxengine/conf /jboxengine/data /jboxengine/data/db`
-4. Configure JuliaBox:
+## Single-machine setup
+
+### Installation overview
```

Comment:
Overview applies to even the cluster setup. Should probably be right under the "Installation" section.